homu/plan.sh

```diff
@@ -0,0 +1,58 @@
+pkg_origin=core
+pkg_name=homu
+pkg_version=0.2.0
+pkg_maintainer='The Habitat Maintainers <humans@habitat.sh>'
+pkg_license=('MIT')
+pkg_description="Homu is a gatekeeper for your commits."
+pkg_upstream_url=http://homu.io/
+pkg_source=nosuchfile.tgz
+pkg_deps=(
+  core/coreutils/8.24
+  core/gcc-libs
+  core/git
+  core/python
+)
+pkg_build_deps=(
+  core/cacerts
+  core/python
+)
+pkg_bin_dirs=(bin)
+pkg_svc_run="bin/homu --verbose"
+pkg_expose=(54856)
+
+do_download() {
+  return 0
+}
+
+do_verify() {
+  return 0
+}
+
+do_unpack() {
+  return 0
+}
+
+do_prepare() {
+  git config --global http.sslCAInfo \
+    "$(pkg_path_for core/cacerts)/ssl/certs/cacert.pem"
+  pyvenv "$pkg_prefix"
```

Comment:
I didn't realize that Python 3 now includes their own type of `virtualenv`. Wow.